tests/testthat/test-github_workflow_present.R

```diff
@@ -23,6 +23,7 @@ test_github_workflow_found <- function(source) {
   )
 }
 
+skip_on_cran()
```

Comment:
not sure this is going to work outside of test that. See the test regional data skipping stuff for something I think does work (maybe)